controllers/pwas/login.js

```diff
@@ -0,0 +1,82 @@
+/**
+ * Copyright 2015-2016, Google, Inc.
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+// const CLIENTID = '605287872172-js6omne47i1k79hnfo7d4bdu9rlemslr.apps.googleusercontent.com';
+// const CLIENTSECRET = 'jc3R9JIRCes-yeEYqjnGKrhL';
+
+const express = require('express');
+const bodyParser = require('body-parser');
```

Comment:
bodyParser is being required and used on [app.js](https://github.com/GoogleChrome/gulliver/blob/master/app.js#L33). Does it need to be included and used again here?